api/v1alpha3/types.go

```diff
@@ -358,6 +358,11 @@ const (
 )
 
 // OSDisk defines the operating system disk for a VM.
+//
+// WARNING: this requires any updates to ManagedDisk to be manually converted. This is due to the odd issue with
```

Comment:
this warning should be in v1alpha4 types

Reply:
I thought it should not be in the hub, but rather each spoke.

Reply:
It's more likely that someone changes a v1alpha4 type which would require them to update conversion since that's the API version we're currently working on. Having the comment in v1alpha3 only means they risk not seeing it if they change api/v1alpha4/types.go

Reply:
added